tox.ini

```diff
@@ -2,72 +2,69 @@
 # See LICENSE file for licensing details.
 
 [tox]
-skipsdist=True
+no_package = True
 skip_missing_interpreters = True
-envlist = fmt, lint, unit
+env_list = lint, unit, integration-{lxd,k8s}-{kafka,mysql,mongodb,postgresql}
 
 [vars]
-src_path = {toxinidir}/src/
-tst_path = {toxinidir}/tests/
-lib_path = {toxinidir}/lib/
-all_path = {[vars]src_path} {[vars]tst_path} {[vars]lib_path}
+src_path = {tox_root}/src
+tests_path = {tox_root}/tests
+lib_path = {tox_root}/lib
+all_path = {[vars]src_path} {[vars]tests_path} {[vars]lib_path}
 
 [testenv]
+allowlist_externals = poetry, env
```

Comment:
oh, no. This was a left-over from testing/debugging. I was putting `env` in the commands (instead of the poetry and running integration tests) only for printing the env and figuring out whether the one I specified were there or not. I'll remove this